asio/include/asio/experimental/impl/coro.hpp

```diff
@@ -606,7 +606,7 @@ struct coro_promise final :
   using allocator_type =
     typename std::allocator_traits<associated_allocator_t<Executor>>::
       template rebind_alloc<std::byte>;
-  using traits = coro_traits<Yield, Return, Executor>;
+  using traits = coro_traits<Yield, Return, Executor, Allocator>;
```

Comment:
``coro_traits`` does not support template parameter ``Allocator``.
https://github.com/chriskohlhoff/asio/blob/a71f5232d207b4f3bbd253eb1041e30b5e4ea606/asio/include/asio/experimental/coro_traits.hpp#L113-L114

Reply:
Oh, how the hell did that compile then?

Reply:
Maybe C++20 does not enabled?